### DISCUSSION TITLE
Failed unpacking rock file

### DISCUSSION THREAD

**Discussion by rumulus** [daurnimator/lua-http#196] [Q&A]
hello @daurnimator i have a problem with the instalation of the http library

im following your step but it doesnt work for me and i dont know why

C:\Users\hulka\Downloads>luarocks install http
Installing http://luarocks.org/repositories/rocks/http-0.4-0.all.rock...

Error: Failed unpacking rock file: C:\Users\hulka\AppData\Local\Temp\luarocks_luarocks-rock-http-0.4-0-2970/http-0.4-0.all.rock

C:\Users\hulka\Downloads>

id be glad if you could shine some light into it

**Comment by daurnimator** [comment]
I'm not sure why it failed; but lua-http doesn't work on windows anyway (due to it's dependency 'cqueues' not working on windows)
